### DISCUSSION TITLE
Deployment status loading forever on PC only

### DISCUSSION THREAD

**Discussion by zacBkh** [vercel/community#2895] [Bug Report]
### Page to Investigate

https://vercel.com/zacbkh/spot-finder/41xauLcTYkWbMREGyC79qfboquge

### Steps to Reproduce

When I push my code and log in to vercel **from my PC** to see the Deployment status (the accordion menu where there is "building", "deployment summary" etc) there is a kind of infinite client side loading/skeleton that prevents me from opening any of the menu items.
First, I thought it was only because my project is fairly large, and deployment was still being processed.

BUT, when I access from my phone, it works perfectly...

I am using chrome on my PC FYI.

![issue vercel](https://github.com/vercel/community/assets/108217390/81b4a6d0-78f2-4ae7-8345-c982a4ee738f)

**Comment by Lewitje** [comment]
Same as this issue #2786
